docs/sts/client-grants.md

```diff
@@ -24,6 +24,14 @@ The OAuth 2.0 access token that is provided by the identity provider. Applicatio
 | *Length Constraints* | *Minimum length of 4. Maximum length of 2048.* |
 | *Required* | *Yes* |
 
+#### Version
+Indicates STS API version information, only supported values is '2011-06-15'.
```

Comment:
"...only supported value is...".  
Also, do you want to include reasoning behind why this is the only supported value?

Reply:
There is no reasoning, it's an API that's the only value.

Reply:
Added reasoning that its for compatibility @eco-minio PTAL